wgsl/index.bs

```diff
@@ -539,29 +531,30 @@ The following types are <dfn dfn noexport>host-shareable</dfn>:
 * [=numeric vector=] types
 * [[#matrix-types]]
 * [[#array-types]] if the array type has a [=stride=] attribute and its element type is host-shareable
-* [[#struct-types]] if each member is host-shareable and has an [=offset=] attribute
+* [[#struct-types]] if each member is host-shareable
 
 <table class='data'>
   <caption>Layout attributes, for host-shareable types</caption>
   <thead>
     <tr><th>Decoraton<th>Operand<th>Description
   </thead>
-
   <tr><td><dfn noexport>`stride`</dfn>
       <td>positive i32 literal
       <td>Applied to an array type.<br>
          The number of bytes from the start of one element of the array to the
          start of the next element.
-  <tr><td><dfn noexport>`offset`</dfn>
-      <td>non-negative i32 literal
+  <tr><td><dfn noexport>`size`</dfn>
+      <td>positive i32 literal
+      <td>Applied to a member of a structure type.<br>
+         The number of bytes used to store this member.
+  <tr><td><dfn noexport>`align`</dfn>
+      <td>positive i32 literal
       <td>Applied to a member of a structure type.<br>
-         The number of bytes between the start of the structure and the location
-         of this member.
+         The member's byte offset from the start of the structure will be a multiple of this value.
 </table>
 
 Note: An [=IO-shareable=] type *T* would also be host-shareable if *T* and its subtypes have
-appropriate [=stride=] and [=offset=] attributes, and if *T* is not [=bool=] and does
-not contain a [=bool=].
+appropriate [=stride=] attributes, and if *T* is not [=bool=] and does not contain a [=bool=].
```

Comment:
if we make this a list, we can omit "and does not contain a ...", since the list would apply to "*T* and its subtypes"

Reply:
It's not exactly clear to me how you visualise this list. Given that comment seems to be related to text prior to this change, do you mind if we try to address this as a separate change or followup?

Reply:
yeah, this is not critical, we can follow up